lib/codelog/output/release_file.rb

```diff
@@ -0,0 +1,17 @@
+module Codelog
+  module Output
+    class ReleaseFile
+      def self.print(content, file_path)
+        new.print(content, file_path)
+      end
+
+      def print(content, file_path)
```

Comment:
@emilianoLeite A way to make the outputters interface the same (and allow dependency injection) would be store the file location directly on this class, but i thought it would make the use of this class more restrict in the future, since the place where the files would be stored would not be dynamic.

Reply:
@Guilherme2112 Maybe you could supply the `file_path` upon initialization, that way `#print` only has to receive the `contents`